File: app/src/main/java/mattframpton/endclothing_mattframpton/model/ProductList.java
package mattframpton.endclothing_mattframpton.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

@SuppressWarnings("unused")
public class ProductList {

    @SerializedName("product_count")
    private Long productCount;
    @Expose
    private List<Product> products;
    @Expose
    private String title;

    public Long getProductCount() {
        return productCount;
    }

    public void setProductCount(Long productCount) {
        this.productCount = productCount;
    }

    public List<Product> getProducts() {
        return products;
    }

    public void setProducts(List<Product> products) {
        this.products = products;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

}

File: app/src/main/java/mattframpton/endclothing_mattframpton/model/ProductList.kt
package mattframpton.endclothing_mattframpton.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ProductList {

    @SerializedName("product_count")
    var productCount: Long? = null
    @Expose
    var products: List<Product>? = null
    @Expose
    var title: String? = null

}
